lib/test/reporters.js

```diff
@@ -22,7 +22,7 @@ ObjectDefineProperties(module.exports, {
     configurable: true,
     enumerable: true,
     get() {
```

Comment:
There’s no need to keep it as a getter, could you modify that?

Reply:
Sure.
Should I do it for all or only for 'spec' property ?